strutil/strutil.go

```diff
@@ -39,6 +39,19 @@ func SizeToStr(size int64) string {
 	panic("SizeToStr got a size bigger than math.MaxInt64")
 }
 
+// SliceToCommaSeparatedString converts an int array to a comma-separated string
+func SliceToCommaSeparatedString(vals []int) string {
```

Comment:
this should be called IntsToCommaSeparated otherwise it's unclear that it takes []int